### PR TITLE
[viewer] fix: empty lists viewer issues

### PR DESCRIPTION
**Pull request by maimonav**
Click below to open examples:
rich-content: https://rich-content-emptyListViewer.surge.sh/
rich-content-storybook: https://rich-content-storybook-emptyListViewer.surge.sh/

### CODE REVIEW

File: packages/viewer/web/src/utils/convertContentState.jsx [hunk 1]
```diff
@@ -13,7 +13,9 @@ import { combineMappers } from './combineMappers';
 import { getInteractionWrapper, DefaultInteractionWrapper } from './getInteractionWrapper';
 
 const isEmptyContentState = raw =>
-  !raw || !raw.blocks || (raw.blocks.length === 1 && raw.blocks[0].text === '');
+  !raw ||
```
Comment (shaulgo) [inline packages/viewer/web/src/utils/convertContentState.jsx:16]: why is this change needed?

Comment (maimonav) [reply packages/viewer/web/src/utils/convertContentState.jsx:16]: It solves the edge case which is when the editor has empty list and the viewer doesn’t show anything, with this change, the viewer gets render with empty list.
